api/events/MSAMEventCollector.yml

```diff
@@ -66,6 +69,14 @@ Parameters:
     AllowedPattern: \S+
     MinLength: 1
     ConstraintDescription: Please enter a value for this field.
+  ContentTableName:
+    Description: >-
+      This is the DynamoDB table name that collects all content for
```

Comment:
Change description to:   This is the DynamoDB table name that stores cached content for MSAM

just to be consistent with the description in all resources template.

Reply:
@morjoan done